tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -137,7 +137,7 @@ def _sync_rbds(self):
                     size=v['size'],
                     pool_id=pool_id,
                     flags=v['flags'],
-                    provisioned=self._to_bytes(v['provisioned']),
+                    provisioned=self._to_bytes(v.get("provisioned", 0)),
```

Comment:
@shtripat if provisioned field is not present then logic  written here also will give wrong value https://github.com/Tendrl/ceph-integration/blob/develop/tendrl/ceph_integration/sds_sync/__init__.py#L393

so setting 0 here is not solve the problem.

if provisioned filed is not there the du command will give used only, but as per loggic we written there will give wrong out put. so this solution will come there i feel.